apikana-maven-plugin/pom.xml

```diff
@@ -26,8 +26,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.6.1</version>
                     <configuration>
-                        <source>1.8</source>
-                        <target>1.8</target>
+                        <source>21</source>
+                        <target>21</target>
```

Comment:
I assume the possible trouble this can have on backwards compatibility are known and appropriate measures are already planned. (I do not have the required insight, so I just assume this is the case).
```patch
-  <target>1.8</target>
+  <target>21</target>
```

Reply:
This PR introduces jakarta.validation into the generated API objects. Therefore, minimal Java Version is 17 anyway, so we might as well go to 21.